### PR TITLE
Test features part 2

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about testing the function responsible for editing the task description, updating the completed status, and cleaning all completed tasks. The following requirements have been done:
- General requirements :heavy_check_mark: 
- JavaScript requirements :heavy_check_mark: 
- Project requirements :heavy_check_mark:

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // Assess
+    expect(result).toEqual(expectedRes);
+  });
+  test('if empty array is returnd passing an empty array as argument', () => {
+    expect(onBlurMocked([])).toEqual([]);
+  });
+  test('if empty array is returnd passing not an array as argument', () => {
+    expect(onBlurMocked('data')).toEqual([]);
+  });
+});
```
Comment (EricMbouwe) [inline index.test.js]: - [ ] These tests are not running. Let's try to that issue

Screenshot from 2021-10-07 12-30-09

File: update.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // Assess
+    expect(changedStateMocked(tempData)).toEqual(expectedRes);
+  });
+  test('if empty array is returnd without passing an argument', () => {
+    expect(changedStateMocked()).toEqual([]);
+  });
+  test('if empty array is returnd passing not an array as argument', () => {
+    expect(changedStateMocked('data')).toEqual([]);
+  });
+});
```
Comment (EricMbouwe) [inline update.test.js]: - [ ]  These tests are not running. Let's try to that issue

Screenshot from 2021-10-07 12-30-55

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @azy64 and @M0rrighan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR
- Good commits
- Good code test coverage

# Required Changes ♻️
- [ ] Please add a section to your Readme for the setup and to run tests.
- [ ] I would recommend adding your tests in a folder to keep your code organized. It was a little hard to find your tests files
- [ ] Only 1 of the tests module is passing. Let's make all the tests green

Screenshot from 2021-10-07 12-33-27


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: update.test.js (hunk 2 above)
Comment (azy64) [reply update.test.js]: test

<h3>:warning:  Thank you for your review but according to the image above all tests passed. Please check it again

</h3>

File: index.test.js (hunk 1 above)
Comment (azy64) [reply index.test.js]: the problem happened only on your computer but on our computers everything is working wel
index-test
update-test
utilities-test

<h1>PLease check it again and  may you fix the problem on your machine </h1>

**Review by mke2111 — APPROVED** [review]
### Hi @azy64, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**